static/src/javascripts/projects/common/modules/experiments/__fixtures__/ab-test.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type { ABTest, Variant } from '../ab-types';
+
+export const genVariant = (id: string): Variant => ({
+    id,
+    test: () => undefined,
+});
+
+export const genAbTest = (id: string, canRun: ?boolean): ABTest => ({
```

Comment:
Perhaps the second parameter could be a (default) object allowing us to pass in options for any of the properties below.